slt/standard/functions/scalar/string/translate.slt

```diff
@@ -0,0 +1,108 @@
+# TRANSLATE function
+
+statement ok
+SET verify_optimized_plan TO true;
+
+# Basic translate functionality
+query T
+SELECT translate('12345', '143', 'ax');
+----
+a2x5
+
+query T
+SELECT translate('hello', 'el', 'XY');
+----
+hXYYo
+
+query T
+SELECT translate('hello world', 'lo', 'XY');
+----
+heXXY wYrXd
+
+# Delete characters when from_set is longer than to_set
+query T
+SELECT translate('hello', 'elo', 'X');
+----
+hX
+
+query T
+SELECT translate('hello', 'helo', '');
+----
+(empty)
+
+query T
+SELECT translate('abcdef', 'ace', 'XY');
+----
+XbYdf
+
+# Empty strings
+query T
+SELECT translate('', 'abc', 'xyz');
+----
+(empty)
+
+query T
+SELECT translate('hello', '', '');
+----
+hello
+
+query T
+SELECT translate('hello', 'abc', '');
+----
+hello
+
+# Unicode support
+query T
+SELECT translate('héllo', 'é', 'e');
+----
+hello
+
+query T
+SELECT translate('🌍🌎🌏', '🌍🌏', 'AB');
+----
+A🌎B
+
+# NULL handling
+query T rowsort
+SELECT translate(input_text, 'abc', 'xyz') FROM (VALUES ('hello'), (NULL), ('abc')) v(input_text);
+----
+NULL
+hello
+xyz
+
+query T rowsort
+SELECT translate('hello', from_chars, 'XYZ') FROM (VALUES ('abc'), (NULL), ('el')) v(from_chars);
+----
+NULL
+hXYYo
+hello
+
+query T rowsort
+SELECT translate('hello', 'el', to_chars) FROM (VALUES ('XY'), (NULL), ('AB')) v(to_chars);
+----
+NULL
+hABBo
+hXYYo
```

Comment:
Remove rowsorts, add an extra 'id' column to the values output and order by that.